criu/cr-restore.c

```diff
@@ -1781,10 +1781,13 @@ static int create_children_and_session(void)
 static int restore_task_with_children(void *_arg)
 {
 	struct cr_clone_arg *ca = _arg;
+	struct pstree_item *parent;
+	u32 parent_cg_set;
```

Comment:
Why is this not initialized? To `0` perhaps.

Reply:
Good catch! Thank you.